cli.js

```diff
@@ -8,6 +8,7 @@ var argv = require('optimist')
 .describe('cross-platform', 'do not archive platform-specific files in node_modules')
 .describe('incremental-install', 'start npm install with last node_modules instead of clearing them')
 .describe('production', 'start npm install with production flag')
+.describe('fullText', 'Sha-1 calculated from full text of package.json')
```

Comment:
I think `check-all-json-elements` will be better. What's your opinion?
If you agree, please check the new PR. Thanks.